crypto/dsa/ecdsa/secp256k1.go

```diff
@@ -96,7 +95,6 @@ func SECP256K1BytesToPublicKey(input []byte) (jwk.JWK, error) {
 	}
 
 	return jwk.JWK{
-		ALG: SECP256K1JWA,
```

Comment:
I ended up removing these because more often than not, the examples I'm seeing are missing the `alg`

Reply:
awesome. left them off for the same reason